pkg/app/server/service/apiservice/service.proto

```diff
@@ -150,6 +151,18 @@ message DisableApplicationResponse {
     string application_id = 1 [(validate.rules).string.min_len = 1];
 }
 
+message UpdateApplicationRequest {
+    string application_id = 1 [(validate.rules).string.min_len = 1];
+    string piped_id = 2 [(validate.rules).string.min_len = 1];
+    string platform_provider = 3 [(validate.rules).string.min_len = 1];
+    model.ApplicationGitPath git_path = 4 [(validate.rules).message.required = true];
+    bool disabled = 5;
```

Comment:
Looks like this is redundant since we have Disable/Enable Application rpc already 👀

Reply:
Surely, I delete disabled field.